### DISCUSSION TITLE
What is the RT-Smart status for RISC-V?

### DISCUSSION THREAD

**Discussion by fanghuaqi** [RT-Thread/rt-thread#8243] [Q&A]
Hello, maintainers!

I would like to ask about status of RT-Smart for RISC-V?

1. Is it run on S-Mode?
2. Is it able to run on RV32 with SV32 MMU?
3. Is it able to run on RV64 with SV32/SV48 MMU?
4. Is there any evaluation bsp can try RT-Smart?

Thanks

**Comment by geniusgogo** [comment]
老早之前rt-smart可以在rv64上运行，可以参考D1/D1S：https://github.com/RT-Thread/rt-thread/tree/master/bsp/allwinner

后来由于工具链更新，并支持了动态加载之后，RV64上还尚未支持，所以暂时应该是在应用程序这块不支持。但是内核态应该可以运行。

**Reply by fanghuaqi** [reply]
Thank you
